spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -156,6 +156,9 @@
           threatmetrix_success: true,
           threatmetrix_request_id: threatmetrix_request_id,
         )
+        proofing_component = ProofingComponent.first
```

Comment:
can we make sure it's the user's proofing component?

```suggestion
        proofing_component = user.proofing_component
```